docs/content/en/hugo-pipes/introduction.md

```diff
@@ -20,12 +20,46 @@ aliases: [/assets/]
 
 Asset files must be stored in the asset directory. This is `/assets` by default, but can be configured via the configuration file's `assetDir` key.
 
-### From file to resource
+### From file or URL to resource
 
-In order to process an asset with Hugo Pipes, it must be retrieved as a resource using `resources.Get`, which takes one argument: the filepath of the file relative to the asset directory.
+In order to process an asset with Hugo Pipes, it must be retrieved as a resource using `resources.Get`. The first argument can be the filepath of the file relative to the asset directory or the URL of the file.
 
 ```go-html-template
 {{ $style := resources.Get "sass/main.scss" }}
+{{ $remoteStyle := resources.Get "https://www.example.com/styles.scss" }}
+```
+
+When using an URL the functions accepts [variadic arguments][variadic]:
+
+```
+{{ $resource := resources.Get "url" "arg1" "arg2" "arg n" }}
```

Comment:
I will look at the complete PR (in general, it looks very good), so you don't need to do something about this right away -- but I write it now while remembering it:

The original `getJSON` etc. API with variadic arguments was a mistake. If people want to concatenate and URL from different parts, they can do so before passing it to `resources.Get`.

So, I suggest that we stick with the 1 argument, with an optional second argument for HTTP headers. Thinking about this, maybe we should make this a `options` so we could add stuff to it in the future (e.g. HTTP config, timeout, whatever), so we do `options.headers` for now?

Reply:
Agree, this would be a better approach, I wanted to stick with what is used elsewhere but as this will deprecate the other functions it's a good moment to break that logic.

I have seen some issues asking to support other HTTP methods (e.g., POST), we might be able to include support for that (including the corresponding payload) as well.

Reply:
OK, I have looked at this one more time -- and if you do the one change as we discussed above, I will merge it (we/I will improve the cache situation in an upcoming commit).

So, we make it into 1 URL argument with an optional second `options` argument. 

```
{{ $resource := resources.Get "https://example.org/api" (dict "headers" (.dict "Authorization" "Bearer abcd" ))  }}
```